templates/eap-online-image-streams.json

```diff
@@ -0,0 +1,69 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-online-image-streams",
+        "annotations": {
+            "description": "ImageStream definitions for Red Hat JBoss Enterprise Application Platform Online.",
+            "openshift.io/provider-display-name": "Red Hat, Inc."
+        }
+    },
+    "items": [
+    {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "eap-online-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP Online",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "12.0"
+                }
+            },
+            "labels": {
+                "xpaas": "1.4.9"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "12",
+                        "annotations": {
+                            "description": "The latest available build of JBoss EAP Online Tech Preview.",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eap:7.2,javaee:7,java:8",
+                            "sampleRepo": "https://github.com/jboss-developer/jboss-eap-quickstarts.git",
+                            "sampleContextDir": "kitchensink",
+                            "sampleRef": "7.1",
+                            "version": "12",
+                            "openshift.io/display-name": "Red Hat JBoss EAP Online (Tech Preview)"
+                        },
+                        "from": {
+                            "kind": "ImageStreamTag",
+                            "name": "12.0"
+
+                        }
+                    },
+                    {
+                        "name": "12.0",
+                        "annotations": {
+                            "description": "JBoss EAP Online Tech Preview S2I Image",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eap:7.2,javaee:7,java:8",
+                            "sampleRepo": "https://github.com/jboss-developer/jboss-eap-quickstarts.git",
+                            "sampleContextDir": "kitchensink",
+                            "sampleRef": "7.1",
+                            "version": "12.0",
+                            "openshift.io/display-name": "Red Hat JBoss EAP Online (Tech Preview)"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "eap-online-openshift:12.0"
```

Comment:
This should probably be: `jboss-eap-7/eap72-online-openshift:12.0` or whatever's been agreed upon by your team.

Reply:
Yeah, good point.